src/services/news/index.jsx

```
@@ -0,0 +1,314 @@
+        )}
+      </Form>
+    </>
+  );
+};
+
+export const NewsPreviewer = ({ image, isOpen, onClose, isTop = false }) => {
+  const dispatch = useDispatch();
+  const { user } = useSelector((state) => state.user);
+  const { token } = useSelector((state) => state.auth);
+  const { isLoading } = useSelector((state) => state.onnews);
+  const { textColor } = useSelector((state) => state.theme);
```

ESLint

[error] 'textColor' is assigned a value but never used.

(no-unused-vars)